src/expandableCalendar/index.tsx

```diff
@@ -562,6 +566,7 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
             horizontal={horizontal}
             firstDay={firstDay}
             calendarStyle={calendarStyle}
+            // TODO: others might container unwanted props, we should use extractCalendarListProps
             {...others}
```

Comment:
Actually the 'others' here are already extracting all the Expandable's props and more